extensions/wikia/PhalanxII/tests/PhalanxModelTest.php

```diff
@@ -224,189 +224,181 @@ public function phalanxNewFromTypeProvider() {
 		/* invalid user */
-		$invalidUserEmail = array(
-			'isAnon'    	=> true,
-			'getName'   	=> self::INVALID_USERNAME,
-			'email'			=> self::INVALID_EMAIL,
-			'block'     	=> (object) array(
+		$invalidUserEmail = [
+			'isAnon' => true,
+			'getName' => self::INVALID_USERNAME,
+			'email' => self::INVALID_EMAIL,
+			'block' => (object) [
 				'regex' => 0,
 				'expires' => '',
 				'text' => self::INVALID_EMAIL,
 				'reason' => 'Test Email',
 				'exact' => '',
 				'caseSensitive' => '',
 				'id' => 4010,
-				'language' => '',
 				'authorId' => 184532,
-			),
-			'result'    	=> 0,
-			'error'			=> wfMsg( 'phalanx-user-block-new-account' )
-		);
+			],
+			'result' => 0,
+			'error' => wfMsg( 'phalanx-user-block-new-account' )
+		];
 
 		/* phalanxexempt */
-		$okUser = array(
-			'isAnon'    => false,
-			'getName'   => self::VALID_USERNAME,
-			'email'		=> self::VALID_EMAIL,
-			'block'     => 0,
-			'result'    => 1,
-			'error'		=> ''
-		);
-
-		return array( $validUser, $invalidUser, $invalidUserEmail, $okUser );
+		$okUser = [
+			'isAnon' => false,
+			'getName' => self::VALID_USERNAME,
+			'email' => self::VALID_EMAIL,
+			'block' => 0,
+			'result' => 1,
+			'error' => ''
+		];
+
+		return [ $validUser, $invalidUser, $invalidUserEmail, $okUser ];
 	}
 
-	public function phalanxTextModelDataProvider() {
+	public function phalanxTextModelDataProvider(): array {
 		/* valid text */
-		$validWiki = array(
-			'text'		=> self::VALID_WIKIA_NAME,
-			'block'     => 0,
-			'result'    => 1,
-		);
+		$validWiki = [
+			'text' => self::VALID_WIKIA_NAME,
+			'block' => 0,
+			'result' => 1,
+		];
 
 		/* invalid text */
-		$invalidWiki = array(
-			'title'		=> self::INVALID_WIKIA_NAME,
-			'block'     => (object) array(
+		$invalidWiki = [
+			'title' => self::INVALID_WIKIA_NAME,
+			'block' => (object) [
 				'regex' => 0,
 				'expires' => '',
 				'text' => self::INVALID_WIKIA_NAME,
 				'reason' => 'Test wiki creation block',
 				'exact' => '',
 				'caseSensitive' => '',
 				'id' => 4013,
-				'language' => 'en',
 				'authorId' => 184532,
-			),
-			'result'    => 0,
-		);
+			],
+			'result' => 0,
+		];
 
 		/* empty text */
-		$invalidWiki = array(
-			'text'		=> '',
-			'block'     => 0,
-			'result'    => 1,
-		);
+		$invalidWiki = [
+			'text' => '',
+			'block' => 0,
+			'result' => 1,
+		];
 
-		return array( $validWiki, $invalidWiki );
+		return [ $validWiki, $invalidWiki ];
 	}
 
-	public function phalanxContentModelDataProvider() {
+	public function phalanxContentModelDataProvider(): array {
 		/* valid textbox & summary */
-		$validContent = array(
-			'title'			=> self::VALID_TITLE,
-			'textbox'		=> self::VALID_CONTENT,
-			'summary'   	=> self::VALID_SUMMARY,
-			'block_text'	=> 0,
-			'block_summary'	=> 0,
-			'result_text'  	=> 1,
-			'result_summary'=> 1
-		);
+		$validContent = [
+			'title' => self::VALID_TITLE,
+			'textbox' => self::VALID_CONTENT,
+			'summary' => self::VALID_SUMMARY,
+			'block_text' => 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "block_text" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)